docs/content/modeling/advanced/entitlements.mdx

```diff
@@ -353,7 +354,7 @@ Add it now. Like so:
 
 In this tutorial, you will find the phrases <ProductConcept section="what-are-direct-and-implied-relationships" linkName="direct relationship and implied relationship" />.
 
-A _direct relationship_ R between user X and object Y means the relationship tuple (user=X, relation=R, object=Y) exists, and the <ProductName format={ProductNameFormat.ShortForm}/> authorization model for that relation allows this direct relationship (by use of `self`).
+A _direct relationship_ R between user X and object Y means the relationship tuple (user=X, relation=R, object=Y) exists, and the <ProductName format={ProductNameFormat.ShortForm}/> authorization model for that relation allows this direct relationship (by use of [direct relationship type restriction](../../configuration-language.mdx#the-direct-relationship-type-restrictions)).
```

Comment:
I _think_ we use the plural `direct relationship type restrictions` instead of the singular `direct relationship type restriction` to refer to the concept in most other places in the docs. I might be misreading this - forgive me if this is contextually accurate and I'm making a mistake.

```suggestion
A _direct relationship_ R between user X and object Y means the relationship tuple (user=X, relation=R, object=Y) exists, and the <ProductName format={ProductNameFormat.ShortForm}/> authorization model for that relation allows this direct relationship (by use of [direct relationship type restrictions](../../configuration-language.mdx#the-direct-relationship-type-restrictions)).
```